cli/Sources/Noora/Noora.swift

```diff
@@ -515,6 +563,65 @@ public class Noora: Noorable {
         return component.run()
     }
 
+    public func singleChoicePrompt<T>(
+        title: TerminalText?,
+        question: TerminalText,
+        options: [T],
+        description: TerminalText?,
+        collapseOnSelection: Bool,
+        filterMode: SingleChoicePromptFilterMode,
+        autoselectSingleChoice: Bool,
+        cancelKey: Character,
```

Comment:
I'd expect us to provide a default, sensible choice, instead of forcing the developer to pick the cancel key.